app/controllers/async_request/jobs_controller.rb

```diff
@@ -1,10 +1,12 @@
+require 'jwt'
+
 module AsyncRequest
   class JobsController < ActionController::Base
     def show
       return render_invalid_token unless valid_token?
-      job = Job.find_by(token[:job_id])
+      job = Job.find_by_id(token[:job_id])
```

Comment:
find_by(id: ...)

Reply:
What's the difference?

Reply:
Why not just find ?

Reply:
Because I don't want it to raise an exception.

Reply:
Rubocop should raise an offense for dynamic finders [see this](https://github.com/bbatsov/rails-style-guide#find_by). Since dynamic finders are generated via method_missing there is also a little performance issue, so if there is a faster version and equally readable why not use it?